src/FsCheck.Xunit/PropertyAttribute.fs

```diff
@@ -205,6 +213,7 @@ type PropertyTestCase(diagnosticMessageSink:IMessageSink, defaultMethodDisplay:T
 
                     match xunitRunner.Result with
                           | TestResult.True _ ->
+                            dispose target
```

Comment:
Do understand this correctly, that the code only disposes the `target` if the result is `True`?

Isn't the desired behaviour that the object should always be disposed, regardless of outcome?

Reply:
My mistake. I thought xunit only called dispose after a successful test, but it turns out I was wrong. I'll amend amend this to always dispose